src/scenes/home/header/burger/burger.css

```diff
@@ -0,0 +1,21 @@
+.burger {
+  display: none;
+}
+
+@media screen and (max-width: 950px) {
+.burger {
+  display: block;
+  position: absolute;
+  right: 0;
+  margin: 0;
+  margin-right: 7px;
```

Comment:
^ Could be replaced with a one-liner declaring the margin for top/right/bottom/left: 
`margin: 0 7px 0 0`
or, could just get rid of the rule `margin: 0` altogether (since it looks like you just need the margin-right)


That way there's not two rules for margin, with `margin-right` overriding the previous rule of `margin: 0`